dts/bindings/spi/intel,intel-spi.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2018 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Intel SPI Controller
+version: 0.1
+
+description: >
+     This is a representation of the Intel SPI controller.
```

Comment:
We need to add a bit more than just 'Intel SPI Controller' so someone has a bit of clue when to use this binding/compatible.

Reply:
Why should this binding be more precise while other are exactly the same? (I can add galileo in the mix, but that's not really the place)